playback/src/player.rs

```diff
@@ -1,10 +1,10 @@
 use futures::sync::oneshot;
 use futures::{future, Future};
+use futures::sync::mpsc::{UnboundedReceiver, UnboundedSender, unbounded};
```

Comment:
Given that both `std::sync::mpsc` and `futures::sync::mpsc` are used in this file, I'd recommend using the fully qualified path everytime.